content/en/contribute/_index.md

```diff
@@ -11,24 +11,26 @@ There are many ways to contribute to the Community Health Toolkit.
 ### Are you a developer?
 
 The [Development](https://forum.communityhealthtoolkit.org/c/support/development/7) section of the forum is a great place to introduce yourself and ask questions. Or you can also jump right in:
-- Issues labeled [help wanted](https://github.com/medic/cht-core/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22Help%20wanted%22) are a great place to start. 
-- Learn to [build and host apps]({{% ref "building" %}}), and share your experiences
-- [Share a new idea or question](https://forum.communityhealthtoolkit.org/) on our Forum
+- [Get up to speed]({{% ref "medic/onboarding/technical-resources" %}}) with the technologies and tools used by the CHT. Additionally, the [Technology Radar for CHT Contributors](https://docs.communityhealthtoolkit.org/cht-tech-radar-contributors/) gives an overview of the technologies used to build the CHT tools. 
```

Comment:
Should we move the technical resources out of the "medic" path?

Reply:
Good point! moved.